### PR TITLE
Custom navbar button colour

### PR DESCRIPTION
**Pull request by DanielGordon1**
DONE: 

changed the bg-colour of the home button/link to red

### CODE REVIEW

**Comment by DanielGordon1** [comment]
Hey @valerieschraauwers ive changed the bgcolour of the navbar home button to red.
What do you think?

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <ul class="navbar-nav mr-auto">
       <% if user_signed_in? %>
         <li class="nav-item active">
-          <%= link_to "Home", "#", class: "nav-link" %>
+          <%= link_to "Home", "#", class: "nav-link coloured" %>
```
Comment (valerieschraauwers) [inline app/views/shared/_navbar.html.erb]: Please change this to the root_path :)

**Comment by valerieschraauwers** [comment]
Looks great! 🌮

**Review by valerieschraauwers — APPROVED** [review]

**Comment by DanielGordon1** [comment]
Ok I changed the link, could you merge this PR for me @valerieschraauwers

**Pull request merged**